bluepysnap/circuit_ids.py

```diff
@@ -324,11 +320,11 @@ def __init__(self, index, sort_index=True):
     def __iter__(self):
         """Iterator on the CircuitNodeIds."""
         for index in self.index:
-            yield CircuitNodeId(*index)
+            yield utils.CircuitNodeId(*index)
```

Comment:
I'm not fully convinced that `CircuitNodeId` belongs to `util`.
What if we move `CircuitNodeId`, `CircuitEdgeId` (and maybe also `IDS_DTYPE`) to a different file, for example `circuit_ids_types.py`?

Reply:
> I'm not fully convinced that CircuitNodeId belongs to util.
Yeah, that was what I was contemplating too, but I didn't feel the functions belong to `circuit_ids.py` either nor did I want to introduce another file. 

In retrospect, I think another file is cleaner. I'll change it.

Reply:
Done